install/tests/main.tf

```diff
@@ -77,11 +77,10 @@ module "aks" {
 
   domain_name              = "${var.TEST_ID}.${var.domain}"
   enable_airgapped         = false
-  enable_external_database = true
-  enable_external_registry = true
-  enable_external_storage  = true
-  dns_enabled              = true
-  workspace_name           = var.TEST_ID
+  create_external_database = true
+  create_external_registry = true
+  create_external_storage  = true
+  resource_group_name      = "p${var.TEST_ID}"
```

Comment:
**Question:** Why are we prefixing the resource group with `p`?

Reply:
This is a temporary hack because some resources did not accept names starting with a digit and test id can sometime start with number. This file is being removed in a future PR, using the single-cluster modules directly. Hence this will be removed.

Reply:
Got it! 🚢